.credo.exs

```diff
@@ -170,7 +170,7 @@
         {Credo.Check.Refactor.DoubleBooleanNegation, []},
         {Credo.Check.Refactor.ModuleDependencies, false},
         {Credo.Check.Refactor.NegatedIsNil, []},
-        {Credo.Check.Refactor.PipeChainStart, []},
+        {Credo.Check.Refactor.PipeChainStart, false},
```

Comment:
Is this really needed as [per the example](https://github.com/rrrene/credo/blob/master/lib/credo/check/refactor/pipe_chain_start.ex), this is actually nicer to read

Reply:
Reverted in https://github.com/nimblehq/elixir-templates/pull/37/commits/8ef2ac4d2dfcb4ac4f9bd1a657aea819c847fdb2